### PR TITLE
Allow use of intrinsic function for http api authorizers

### PR DESCRIPTION
**Pull request by fredericbarthelet**
Closes: #8135

### CODE REVIEW

File: lib/plugins/aws/provider/awsProvider.js [hunk 1]
```diff
@@ -151,6 +151,9 @@ class AwsProvider {
               { $ref: '#/definitions/awsCfFunction' },
             ],
           },
+          awsString: {
```
Comment (medikoo) [inline lib/plugins/aws/provider/awsProvider.js:154]: Maybe let's name it `awsCfAddress` or `awsCfInstruction` (which is better?), at least _string_ suggests it's unconditionally _string_ (e.g. we have `awsArnString` and it's unconditionally an ARN string)

**Review by medikoo — CHANGES_REQUESTED** [review]
Thank you @fredericbarthelet! Looks good, I have just one naming improvement proposal

File: lib/plugins/aws/provider/awsProvider.js (hunk 1 above)
Comment (fredericbarthelet) [reply lib/plugins/aws/provider/awsProvider.js]: Let's go with `awsCfInstruction`

**Comment by fredericbarthelet** [comment]
@medikoo I renamed the definition with `awsCfInstruction`, WDYT ?

**Review by medikoo — APPROVED** [review]
> @medikoo I renamed the definition with awsCfInstruction, WDYT ?

Sounds good!

Thank you @fredericbarthelet !

**Pull request merged**